static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -95,7 +94,9 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
             </Row>
 
             <TextOverflow style={{gridArea: 'user'}}>
-              <FeedbackItemUsername feedbackIssue={feedbackItem} detailDisplay={false} />
+              <strong>
+                {feedbackItem.metadata.name ?? feedbackItem.metadata.contact_email}
```

Comment:
might need an additional fallback to `Anonymous User` here:
<img width="418" alt="SCR-20240411-itpe" src="https://github.com/getsentry/sentry/assets/56095982/c9383ad7-0208-46df-b3e7-8e58cca7fe89">

Reply:
yeah, i'll write up some tests too.